iib/workers/tasks/build_regenerate_bundle.py

```diff
@@ -529,3 +540,31 @@ def _apply_repo_enclosure(bundle_metadata, org_enclose_repo_namespace, org_enclo
         replacement_pullspecs[pullspec] = new_pullspec
 
     _replace_csv_pullspecs(bundle_metadata, replacement_pullspecs)
+
+
+def _get_related_bundle_images(bundle_metadata, request_id):
```

Comment:
This shouldn't be called `get` if it doesn't return anything. Given what it does, this should probably be called `_write_related_bundles_file`